CHANGES.md

```diff
@@ -1,3 +1,66 @@
+## edge-19.4.5
+
+**Significant Update**
+
+As of this edge release the proxy auto-injector component is always installed.
+To have the auto-injector inject a pod you still have the option to manually
+add the `linkerd.io/inject: enable` annotation into the pod spec, or at the
+namespace level to have all your pods be injected by default.
+With this release the behaviour of the `linkerd inject` command changes, as
+now it won't output the injected sidecar, but instead it will just add the
+annotation to defer the injection to the auto-injector.
+For advanced CI/CD pipelines where you'd like to keep the old behaviour, you can
```

Comment:
I think we should omit the _advanced CI/CI pipelines_, as the `--manual` flag will be relevant to other uses cases like helm, kustomize etc. How about something like, _For use cases that require the full injected YAML to be output, a new `--manual` flag has been added_?